apps/angular/simple-animations/src/app/app.component.ts

```diff
@@ -19,7 +21,7 @@ import { Component } from '@angular/core';
   `,
   template: `
     <div class="mx-20 my-40 flex gap-5">
-      <section>
+      <section [@blockEnter]>
```

Comment:
i think the square brackets here are not necessary -> `@blockEnter` is sufficient